andela/travel_tool_front
# #160371227 A requester should be able to view approved request

#### What does this PR do?
Adds functionality to allow users view approved request on the notification pane.

#### Description of Task to be completed?
- add check to detect if notification is an approval notification and render the `view details` option
- update test to accommodate new changes

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory and run make start
- Login into the application
- Create a new request and ensure you add yourself as the requesters manager
- Navigate to `My Approvals` page and approve the just created request
- Open the notification pane.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[ #160371227](https://www.pivotaltracker.com/story/show/160371227)

#### Screenshots (if appropriate)
<img width="358" alt="screen shot 2018-09-24 at 4 27 42 pm" src="https://user-images.githubusercontent.com/23608060/45961608-cdd67e80-c016-11e8-9c9a-306acac513fc.png">


#### Questions:
